ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+<!-- The title above should be a short description of the issue. -->
+
+#### What is the problem?
```

Comment:
`Brief description of the problem`

Reply:
@smichr Is it ok to write this?

Reply:
I liked the question because it prompts the reporter to answer a question and they might start with the most pertinent description and then get into details.

+0 

"Better than nothing is something that can become anything else." 
\c

Reply:
@sylee957 Please provide your views on [the above comment](https://github.com/sympy/sympy/pull/16742#discussion_r279193997).